### PR TITLE
Move `zoom-frm` package locally since it disappeared from elpa

### LINKED ISSUE #115: "error: Package ‘zoom-frm-20170125.1953’ is unavailable" during install
```
Warning (initialization): An error occurred while loading ‘/home/pholisma/.emacs.d/init.el’:

error: Package ‘zoom-frm-20170125.1953’ is unavailable

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the ‘--debug-init’ option to view a complete error backtrace.
```
I'm running Emacs 25.3.2 on Ubuntu 14.04.

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this?

This PR should fix #115. The issue is the `zoom-frm` has disappeared from elpa & melpa. I'm not quite sure how I feel about copy & pasting the package source into our project.

~I'd like to figure out why this package has disappeared from the package sources.~

Turns out it was a wiki package & melpa removed wiki sourced packages https://github.com/melpa/melpa/pull/5008

### CODE REVIEW

**Comment by cowboyd** [comment]
can we put it in `vendor/zoom-frm.el` with a note about why this was extracted?

**Comment by Robdel12** [comment]
@cowbyd absolutely. 

How hard would it be start building this on circle CI?

**Comment by cowboyd** [comment]
Honestly since Circle 2.0 supports BYOC (bring your own container) It should be a lot easier. We can just have a container for each version of emacs we want to build against.

**Comment by Robdel12** [comment]
Oh my, we should absolutely do that

**Comment by Robdel12** [comment]
@cowboyd I moved the file to `vendor`.

File: frontmacs-keys.el [hunk 1]
```diff
@@ -17,6 +17,7 @@
 (require 'comment-dwim-2)
 (require 'browse-kill-ring)
 (require 'counsel)
+(require 'zoom-frm)
```
Comment (cowboyd) [inline frontmacs-keys.el]: How does `vendor` get added to the load-path? Wouldn't this need to be changed in `frontside.el` or `frontside-config.el`?

Comment (Robdel12) [reply frontmacs-keys.el]: That was going to be my next question. I'll update

Comment (cowboyd) [reply frontmacs-keys.el]: Does it work when you run init-test?

**Pull request merged**